R/FilterState.R

```diff
@@ -254,7 +255,7 @@ FilterState <- R6::R6Class( # nolint
     #' @return NULL invisibly
     #'
     set_keep_na = function(value) {
-      checkmate::assert_flag(value)
+      checkmate::assert_flag(value, null.ok = TRUE)
```

Comment:
Hmm, why is it eased? I think we should limit to TRUE/FALSE only.

Reply:
When the filter is disabled, the state values are set to `NULL`. So we need this here to allow for that.